Shared/Strings/Strings.swift

```diff
@@ -276,6 +304,8 @@ internal enum L10n {
   internal static let emptyNextUp = L10n.tr("Localizable", "emptyNextUp", fallback: "Empty Next Up")
   /// Enabled
   internal static let enabled = L10n.tr("Localizable", "enabled", fallback: "Enabled")
+  /// Supply a custom display name or leave empty to use the name reported by the device.
+  internal static let enterCustomDeviceName = L10n.tr("Localizable", "enterCustomDeviceName", fallback: "Supply a custom display name or leave empty to use the name reported by the device.")
```

Comment:
I want to reduce "leave x in a certain value". Instead, let's have a toggle for using a custom name and we will update the name accordingly.